group_project/assignment_6.py

```diff
@@ -1,15 +1,32 @@
 # Example. Use it as a base to build your own template.
 BOARD_TEMPLATE = """
-O  |  O  |  X
+{}  |  {}  |  {}
 --------------
-X  |  X  |  O
+{}  |  {}  |  {}
 --------------
-O  |  X  |  O
+{}  |  {}  |  {}
 """
 
-
 def format_tic_tac_toe_board(first_row, second_row, third_row):
-    pass
+    r1_1 = first_row[0]
+    r1_2 = first_row[1]
+    r1_3 = first_row[2]
+    
+    r2_1 = second_row[0]
+    r2_2 = second_row[1]
+    r2_3 = second_row[2]
+    
+    r3_1 = third_row[0]
+    r3_2 = third_row[1]
+    r3_3 = third_row[2]
+    
+    board = BOARD_TEMPLATE.format(
+        r1_1, r1_2, r1_3,
+        r2_1, r2_2, r2_3,
+        r3_1, r3_2, r3_3,
+    )
+    
+    return board
 
```

Comment:
Only issues I see is using the Global variable of BOARD_TEMPLATE instead of building your board in your function. Either way, If you go that route, you don’t need to build your board with r1_1, r1_2, etc. You could simple put:

def format_tic_tac_toe_board(first_row, second_row, third_row):

    board = BOARD_TEMPLATE.format(
        first_row[0], first_row[1], first_row[2],
        second_row[0], second_row[1], second_row[2],
        third_row[0], third_row[1], third_row[2],
    )